sql/catalyst/src/main/java/org/apache/spark/sql/vectorized/ArrowColumnVector.java

```diff
@@ -549,21 +544,18 @@ int getInt(int rowId) {
     }
   }
 
-  private static class IntervalDayAccessor extends ArrowVectorAccessor {
+  private static class DurationAccessor extends ArrowVectorAccessor {
 
-    private final IntervalDayVector accessor;
-    private final NullableIntervalDayHolder intervalDayHolder = new NullableIntervalDayHolder();
+    private final DurationVector accessor;
```

Comment:
do we need to care about precision here? (microsecond)

Reply:
oh, so it's just an int64 physically, and the type annotation (or logical type) decides its semantic